src/components/QuickOpenModal.js

```diff
@@ -335,9 +335,7 @@ export class QuickOpenModal extends Component<Props, State> {
     if (newQuery === "") {
       return results;
     }
-    if (Object.keys(MODIFIERS).includes(query[0])) {
-      newQuery = query.slice(1, query.length);
-    }
+    newQuery = query.replace(/#|@|:|\?/gi, " ");
```

Comment:
We should probably use the `MODIFIERS` here, consider the case when we add another modifier, then this code has to be changed and it'd be hard to guess.